openshift.spec

```diff
@@ -2,13 +2,13 @@
 %global debug_package %{nil}
 %global gopath      %{_datadir}/gocode
 %global import_path github.com/openshift/origin
-%global commit      d3f40eafae8ae7bbca61981e33f384375307fafa
+%global commit      21fb40637c4e3507cca1fcab6c4d56b06950a149
```

Comment:
Yeah, anything to keep us from having to manually modify the commit. :)  Thanks for sync'ing with Clayton on this.